core/setup.py

```diff
@@ -38,17 +38,7 @@ def read(fname):
     author_email="info@fishtownanalytics.com",
     url="https://github.com/fishtown-analytics/dbt",
     packages=find_namespace_packages(include=['dbt', 'dbt.*']),
-    package_data={
-        'dbt': [
-            'include/index.html',
-            'include/global_project/dbt_project.yml',
-            'include/global_project/docs/*.md',
-            'include/global_project/macros/*.sql',
-            'include/global_project/macros/**/*.sql',
-            'include/global_project/macros/**/**/*.sql',
-            'py.typed',
-        ]
-    },
+    include_package_data = True,
```

Comment:
A couple articles I read said to use `include_package_data` instead of `package_data` with the MANIFEST.in file so that's why I made this change. You don't have to list the files then.

Example:
https://newbedev.com/how-include-static-files-to-setuptools-python-package

Reply:
I really like it! We did the same in dbt-spark a few months ago: https://github.com/fishtown-analytics/dbt-spark/pull/151. To quote a wise person, "This is much easier."

Reply:
🙏